lib/index.ts

```diff
@@ -20,7 +21,7 @@ const vueSpatialNavigation = {
     // Vue.prototype.$SpatialNavigation = SpatialNavigation;
     Vue.config.globalProperties.$SpatialNavigation = SpatialNavigation;
 
-    const assignConfig = (sectionId, config) => {
+    const assignConfig = (sectionId: string | undefined, config: object) => {
```

Comment:
config is optional. And should be the type of `vjsnOptions` if im not mistaken

Reply:
You're right, fixed it! 🔥